app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+import datetime
+from typing import List, Dict, Any
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: List[Dict[str, Any]], cafe: Cafe) -> str:
```

Comment:
Code Style: It's a good practice to use type annotations. They make your code more understandable. You've done it for the function return type and the `friends` parameter, but you missed it for the `cafe` parameter. It should be annotated as `Cafe`.